README.md

```diff
@@ -197,6 +197,25 @@ WavefrontSender wavefrontSender = new WavefrontClient.Builder(proxyServerWithPor
         build(); // Returns a WavefrontClient
 ```
 
+When you use a Sender SDK, you won’t see span-level RED metrics by default unless you use the Wavefront proxy and define a custom tracing port (`tracingPort`). See [Instrument Your Application with Wavefront Sender SDKs](https://www.wavefront.com/tracing_instrumenting_frameworks.html/#instrument-your-application-with-wavefront-sender-sdks) for details.
```

Comment:
this is relevant only for spans. Can you add a subsection title that this is related to spans?